frontend/pages/create.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { mutate } from "swr";
+import { useRouter } from "next/router";
+import { useAuth, withLoginRequired } from "use-auth0-hooks";
+import RegistryIcon from "@iconscout/react-unicons/icons/uil-diary";
+import AdminPage from "../components/AdminPage";
+import RegistryForm from "../components/RegistryForm";
+import { fetchIt } from "../utils";
+import { AUTH0_API_IDENTIFIER } from "../utils";
+
+const Create = () => {
+  const { push } = useRouter();
+  const { accessToken } = useAuth({ audience: AUTH0_API_IDENTIFIER });
+
+  const onSubmit = formData => {
+    mutate("/registry/admin", async () => {
+      const registry = await fetchIt(`/registry`, {
```

Comment:
probably a good idea to add some try and catch here

don't want to return an error as a registry